README.md

```diff
@@ -159,42 +267,190 @@ You can find other ICU releases at
 Download the file named something like `icu4c-**##.#**-src.tgz` (or
 `.zip`).
 
-Unix/Macintosh: from an already-unpacked ICU
+Unix / Macintosh
 
-```sh
-./configure --with-intl=[small-icu,full-icu] --with-icu-source=/path/to/icu
-```
+```text
+# from an already-unpacked ICU:
+$ ./configure --with-intl=[small-icu,full-icu] --with-icu-source=/path/to/icu
 
-Unix/Macintosh: from a local ICU tarball
+# from a local ICU tarball
+$ ./configure --with-intl=[small-icu,full-icu] --with-icu-source=/path/to/icu.tgz
 
-```sh
-./configure --with-intl=[small-icu,full-icu] --with-icu-source=/path/to/icu.tgz
+# from a tarball URL
+$ ./configure --with-intl=full-icu --with-icu-source=http://url/to/icu.tgz
 ```
 
-Unix/Macintosh: from a tarball URL
+Windows
+
+First unpack latest ICU to `deps/icu`
+[icu4c-**##.#**-src.tgz](http://icu-project.org/download) (or `.zip`)
+as `deps/icu` (You'll have: `deps/icu/source/...`)
 
-```sh
-./configure --with-intl=full-icu --with-icu-source=http://url/to/icu.tgz
+```text
+> vcbuild full-icu
 ```
 
-Windows: first unpack latest ICU to `deps/icu`
-  [icu4c-**##.#**-src.tgz](http://icu-project.org/download) (or `.zip`)
-  as `deps/icu` (You'll have: `deps/icu/source/...`)
+# Building Node.js with FIPS-compliant OpenSSL
```

Comment:
I can confirm that FIPS is not in 0.12

It landed in master in 0f68377 and was never backported